src/main/resources/application.yml

```diff
@@ -114,7 +117,7 @@ app:
     duration: 3600000
 
   # edu.tamu.weaver.auth.service.UserCredentialsService
-  authority.admins:
+  authority.admins: jcreel@tamu.edu, jmicah@tamu.edu, huff@tamu.edu, wwelling@tamu.edu
```

Comment:
Let's leave the emails off of the public repo for now

Reply:
Okay. Removing.